magpie/api/management/user/user_utils.py

```diff
@@ -119,11 +121,21 @@ def _add_to_group(usr, grp):
 
     user_content = uf.format_user(new_user, new_user_groups)
 
+    # Create a token for the tmp_url, in case an error happens in the webhook services
+    token_id = uuid.uuid4()
+    webhook_token = models.TemporaryToken(
+        token=token_id,
```

Comment:
I think the id can be omitted. The model should automatically handle its generation.

Reply:
I did try this at first, but ran into problems. 
If I omit the `token=token_id,` when I create the TemporaryToken, I then try to retrieve the url with `tmp_url = webhook_token.url()` but I get `/tmp/None` as the token has not been generated yet.

I saw just above this code, for the user creation, I think the id also gets generated automatically and after creating a user and submitting it to the database, you retrieve the user again in order to have a 'refreshed" user with its automatically generated id.

Problem here I wasn't able to do this either. I can't retrieve the token since the other fields of the token are not unique (operation, user_id, group_id). I could possibly have multiple tokens that have the same values for those fields, so I would get multiple ids. 

There is also the by_token function which takes the token in parameter, but I can't use that either since it requires the automatically generated token id.
I didn't find another way to retrieve the 'refreshed' token with its automatically generated values.

So I will leave this as it is for now, but I am open to suggestions!